packages/harmony/src/storybook/components/RelatedComponents.tsx

```diff
@@ -72,12 +132,20 @@ const relatedComponentsMap = {
     ),
     description:
       'A layout component to handle flex behavior without manual styling.',
-    link: 'layout-flex--documentation'
+    link: 'components-layout-flex--documentation'
```

Comment:
i see some of the links are prepended with `component-` and some aren't. Do these all work?

Reply:
good catch. This was a bad merge